src/view/drag-handle/drag-handle.jsx

```diff
@@ -192,31 +198,56 @@ export default class DragHandle extends Component<Props> {
 
   onKeyDown = (event: KeyboardEvent) => {
     // let the other sensors deal with it
-    if (this.mouseSensor.isCapturing() || this.touchSensor.isCapturing()) {
+    if (this.mouseSensor.isCapturing() || this.touchSensor.isCapturing() || this.pointerSensor.isCapturing()) {
       return;
     }
-
     this.keyboardSensor.onKeyDown(event);
   };
 
   onMouseDown = (event: MouseEvent) => {
     // let the other sensors deal with it
-    if (this.keyboardSensor.isCapturing() || this.mouseSensor.isCapturing()) {
+    if (this.keyboardSensor.isCapturing() || this.touchSensor.isCapturing() || this.pointerSensor.isCapturing()) {
       return;
     }
 
     this.mouseSensor.onMouseDown(event);
   };
 
   onTouchStart = (event: TouchEvent) => {
-    // let the keyboard sensor deal with it
-    if (this.mouseSensor.isCapturing() || this.keyboardSensor.isCapturing()) {
+    // let the other sensors deal with it
+    if (this.keyboardSensor.isCapturing() || this.mouseSensor.isCapturing() || this.pointerSensor.isCapturing()) {
       return;
     }
-
     this.touchSensor.onTouchStart(event);
   };
 
+  onPointerDown = (event: PointerEvent) => {
+
+    if (!IS_TOUCH_COMPATIBLE) {
```

Comment:
Good optimization here.

Reply:
Although in a non-touch-compatible environment you will end up adding this each time. Could store a separate Boolean for having added the listener.